andela/ah-bird-box
# #163388245 Integrate coveralls code

**What does this PR do?**
This PR Integrates Coveralls code to enable coverage service

**Description of Task to be completed?**
Add coveralls command in .travis.yml file
Add ah-bird-box repo on coveralls.io
Embed coveralls badge on README.md file

**How should this be manually tested?**
- Clone the repo and cd into it
- Do a ```git checkout ch-integrate-coveralls-163388245```
- Check the README.md file for the coveralls badge or
- Check the README.md file under ch-integrate-coveralls-163388245 branch online 

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
[#163388245](https://www.pivotaltracker.com/story/show/163388245)

**Screenshots (if appropriate)**
None
